examples/internal/integration/integration_test.go

```diff
@@ -980,6 +982,27 @@ func testABEList(t *testing.T, port int) {
 	}
 }
 
+func testABEDownload(t *testing.T, port int) {
+	apiURL := fmt.Sprintf("http://localhost:%d/v1/example/download", port)
+	resp, err := http.Get(apiURL)
+	if err != nil {
+		t.Errorf("http.Get(%q) failed with %v; want success", apiURL, err)
+		return
+	}
+	defer resp.Body.Close()
+
+	body, err := readAll(resp.Body)
+	if err != nil {
+		t.Fatalf("readAll(resp.Body) failed with %v; want success", err)
+	}
+
+	want := []string{"Hello 1", "Hello 2"}
+	if !reflect.DeepEqual(body, want) {
+		t.Errorf("testABEDownload failed: got %v, want %v", body, want)
+	}
+
+}
```

Comment:
Could we check that the content type header is correct as well please?

Reply:
added check if content type is correct or not. but not sure how to check content type of each chunk using http client. do you have any idea ?